stories/theme/borderRadius.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { useTheme } from 'emotion-theming';
+import { SPACER } from 'src/constants';
+
+const MainContainer = styled.div`
+  padding: ${SPACER.large} ${SPACER.large};
+`;
+
+export const BorderRadius = () => {
```

Comment:
It would be nice to make this story more like `COLORS`, `BOX_SHADOWS`, etc., that shows the differences in action, rather than just shows the text values.

Reply:
done, also refactored other stories for consistency (i.e. same initial padding in page)